packages/lib/telemetry.ts

```diff
@@ -22,6 +22,7 @@ export const telemetryEventTypes = {
   website: {
     pageView: "website_page_view",
   },
+  slugReplacementAction: "slug_replacement_action",
```

Comment:
What is this for?

Reply:
See https://github.com/calcom/cal.com/pull/6876/files#diff-d75f96ff093473eefda3b50645504e299af40b26538bf2b235dc067293d3af2cR413